vocabularies/UI.xml

```diff
@@ -1370,9 +1370,11 @@ The trend is
         <Annotation Term="Core.IsLanguageDependent" />
       </Term>
 
-      <Term Name="TextArrangement" Type="UI.TextArrangementType" Nullable="false" AppliesTo="Annotation EntityType">
+      <Term Name="TextArrangement" Type="UI.TextArrangementType" Nullable="false" AppliesTo="Annotation">
```

Comment:
@HeikoTheissen if we leave `AppliesTo` untouched we could already merge the PR, and adjust the `AppliesTo` later in an additional PR.